internal/cli/doc-gen.go

```diff
@@ -20,6 +20,13 @@ const (
 
 	parentPageTemplate = `---
 layout: default
+has_toc: false
```

Comment:
**Observation:** One thing I investigated is wether the `has_toc` property needed to be declared, it initially appeared extraneous. It's peculiar that the TOC is opt-out rather than opt-in, just thought that was odd 🤷‍♂️ .